### ISSUE TITLE
Freq axis inconsistent

### ISSUE THREAD

**Issue by newburgh** [WrightLaboratory/newburghlabdrone#19]
I've been having issues with consistency when I run different Fargs, I think the issue is here:

say I try two Fargs: [512,1024,32] and [768,1024,32]:

You define:
Vvals=(np.repeat(np.swapaxes(g_norm[:,fmin:fmax],0,1)[np.newaxis,:,:],len(ccc.inds_on),axis=0)**-1)*ccc.V_bgsub[ccc.inds_on,fmin:fmax,:]           

For my case:
If fmin is 512 and fmax is 1024, vvals is now sized [time,512,chans], with 0 index = 512
if fmin is 768 and fmax is 1024, vvals is now sized [time,256,chans], with 0 index = 768

Then you compute this:
for i,chan in enumerate(range(self.n_channels)):
      for j,fr in enumerate(self.faxis)
                    valsf=Vvals[:,j,i]                        
                    self.V_LC_mean[:,:,j,i,h]=binned_statistic_2d(x=xf,y=yf,values=valsf,statistic='mean',bins=[self.x_edges[:,i],self.y_edges[:,i]]).statistic
      
j is going to be sized my faxis, which is defined by Fargs:
for my first case, j will be 0:16 and fr will be 512, .... 1024
for my second case, j will be 0:8 and fr will be 768 ... 1024

Vvals, however will be called as:
vvals[0:16]: this will be values ranging 512-528
vvals[0:8]: this will be values ranging 768-776

Basically, V_bgsub should grab np.arange(fmin,fmax,fstep) instead of [fmin:fmax]   , I think
